### DISCUSSION TITLE
Layer Editor

### DISCUSSION THREAD

**Discussion by wtelford** [Autodesk/maya-usd#1011] [Announcement]
The USD Layer Editor now lives inside the MayaUSD plugin and is available with the newly compiled [release 0.6.0](https://github.com/Autodesk/maya-usd/releases). This change makes the Layer Editor available to test in Maya 2020.
![2020_layerEditor](https://user-images.githubusercontent.com/1900693/102419123-beabc480-3fc4-11eb-97ec-cd6f11106795.PNG)

**Comment by mikelyndon-ta** [comment]
Hey @wtelford. Are there any docs or a video walkthrough of how to use the layer editor?

**Comment by wtelford** [comment]
Hey Mike,
Currently our USD Plugin Documentation is part of the Maya documentation. As we haven't yet shipped a version of Maya with USD included, that means it is not yet publicly viewable. If you are in the Maya Beta program, please ping me on the Maya beta forums and I can point you to the current docs. 

Until those become publicly available, the most in depth coverage of the layer editor would be in this years Vision series video which I mentioned in this [github discussion](https://github.com/Autodesk/maya-usd/discussions/874).

[Here](https://youtu.be/6EqcwefKMRI?t=1847) is the link to the part of the YouTube video that starts the coverage of the layer editor functionality.
